cou/apps/base.py

```diff
@@ -139,7 +140,11 @@ def _verify_channel(self) -> None:
 
         :raises ApplicationError: Exception raised when channel is not a valid OpenStack channel.
         """
-        if self.is_from_charm_store or self.is_valid_track(self.channel):
+        if (
+            self.is_from_charm_store
+            or self.channel == LATEST_STABLE
+            or self.is_valid_track(self.channel)
+        ):
```

Comment:
It's not responsibility of these PR, but we should move this `_verify_channel` to sanity checks.